test/fixtures/projects.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
was fixtures auto-generated? i don't think we'll really need them if we're not going to use them for testing, so you could remove this file.

but if you meant to do this, could you rename the fixtures so it's more representative of the projects?

Reply:
yeah these were autogenerated. i'll get rid of them